package.json

```diff
@@ -17,7 +17,9 @@
     "coverage": "turbo run coverage",
     "storybook": "turbo run storybook",
     "prepare": "husky install",
-    "release-packages": "turbo run build lint test --filter=./packages/* && changeset publish"
+    "release-packages": "turbo run build lint test --filter=./packages/* && changeset publish",
+    "generate:graphql:fetch:testnet": "turbo run generate:graphql:fetch:testnet",
+    "generate:graphql:fetch:mainnet": "turbo run generate:graphql:fetch:mainnet"
```

Comment:
why do you need these in main package.json too? Is it helper for developer only or also used by CI when deploying?

Reply:
It's just for developers for now and I added it to the main `package.json` so I can call it from there. Do you want me to remove it and call it directly on the frontend dapp only?